content/about/about_people.rst

```diff
@@ -590,6 +590,35 @@ https://github.com/ElijahCaine
 
   <br/>
 
+`Levi Willmeth`_
+----------------
+
+.. image:: /images/lwillmeth.jpg
+    :scale: 50%
+    :align: right
+    :alt: Levi Willmeth
+
+.. class:: no-breaks
+
+  **OSL Academy Intern**
+
+Levi joined the OSL as a student intern in the summer of 2016. He is married
+and has two energetic toddlers who help ensure he doesn't sleep too much.
+Levi enjoys playing bike polo, flying RC aircraft, looking thoughtfully past
```

Comment:
"looking thoughtfully passed" (or maybe beyond if you want to avoid a past vs passed debate altogether. That's what I do).